packages/cogify/src/topo-raster/stac/create-base-stac-item.ts

```diff
@@ -0,0 +1,83 @@
+import { TileMatrixSet } from '@basemaps/geo';
+import { LogType } from '@basemaps/shared';
+import { CliId, CliInfo } from '@basemaps/shared/build/cli/info.js';
+import { GeoJSONPolygon } from 'stac-ts/src/types/geojson.js';
+
+import { MapSheetStacItem } from '../types/map-sheet-stac-item.js';
+import { TiffItem } from '../types/tiff-item.js';
+
+const CLI_DATE = new Date().toISOString();
+
+/**
+ * This function creates a base StacItem object based on the provided parameters.
+ *
+ * @param fileName: The map sheet's filename
+ * @example "CJ10" or "CJ10_v1-00"
+ *
+ * @param tiffItem TODO
+ *
+ * @returns a StacItem object
+ */
+export function createBaseStacItem(
```

Comment:
I believe, the `cover` command have all stac creation logic which are very similar, wonder we can extract them to reuse together.